### PR TITLE
UI: Improve various descriptions in configuration

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do?

Improves text in configuration, based on work of @ybasford

### CODE REVIEW

File: monkey/monkey_island/cc/ui/src/services/configuration/exploiterClasses.js [hunk 1]
```diff
@@ -61,8 +59,8 @@ export const EXPLOITERS = {
             'enum': ['SMBExploiter'],
             'title': 'SMB Exploiter',
             'safe': true,
-            'info': 'Brute forces using credentials provided by user and' +
-            ' hashes gathered by mimikatz.',
+            'info': 'Attempts to brute-force SMB using credentials provided by the user and' +
+            ' hashes gathered by Mimikatz.',
```
Comment (VakarisZ) [inline monkey/monkey_island/cc/ui/src/services/configuration/exploiterClasses.js:63]: A lot of these descriptions are not perfect. Credentials might come from any source, I'm not sure we need to/are able to explain where they come from here

Comment (mssalvatore) [reply monkey/monkey_island/cc/ui/src/services/configuration/exploiterClasses.js:63]: We could say "using all known credentials" or something like that?

**Review by mssalvatore — APPROVED** [review]
Approved, but we should settle on a better wording regarding credentials.

File: monkey/monkey_island/cc/ui/src/services/configuration/exploiterClasses.js (hunk 1 above)
Comment (VakarisZ) [reply monkey/monkey_island/cc/ui/src/services/configuration/exploiterClasses.js:63]: I think that can be implied from the context. Maybe we should just delete the credentials part?

**Comment by ybasford** [comment]
That’s a good idea. Could you delete it?

Yael Basford
Senior Technical Writer

[signature_3632670654]


Office:+1.617.444.1234
Cell:+1.978.868.8785

Akamai Technologies
145 Broadway
Cambridge, MA 02142


Connect with Us:

[signature_1409130784]<https://community.akamai.com/>[signature_3681041544]<http://blogs.akamai.com/>[signature_2795187136]<https://twitter.com/akamai>[signature_3330014064]<http://www.facebook.com/AkamaiTechnologies>[signature_272063351]<http://www.linkedin.com/company/akamai-technologies>[signature_4084934274]<http://www.youtube.com/user/akamaitechnologies?feature=results_main>



From: VakarisZ ***@***.***>
Reply-To: guardicore/monkey ***@***.***>
Date: Thursday, January 12, 2023 at 09:52
To: guardicore/monkey ***@***.***>
Cc: "Basford, Yael" ***@***.***>, Mention ***@***.***>
Subject: Re: [guardicore/monkey] UI: Improve various descriptions in configuration (PR #2819)


@VakarisZ commented on this pull request.

________________________________

In monkey/monkey_island/cc/ui/src/services/configuration/exploiterClasses.js<https://urldefense.com/v3/__https:/github.com/guardicore/monkey/pull/2819*discussion_r1067800080__;Iw!!GjvTz_vk!V6y49G2uxP48zmmBV0KJ_BRhgR6bM_RLVITa24dUCsj2jeUkEzgRf5cj3fDCjZJHbZHWpa0D53kIybocrjtNmWcM$>:

> +            'info': 'Attempts to brute-force SMB using credentials provided by the user and' +

+            ' hashes gathered by Mimikatz.',

I think that can be implied from the context. Maybe we should just delete the credentials part?

**Pull request merged**